### PR TITLE
feat: SUITEDEV-35218 add new fetch contact list method

### PR DESCRIPTION
**Pull request by roxaaams**
### All Submissions:
* [x] Have you followed our guidelines?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New API Endpoints:
* [ ] Is it a _public_ API endpoint that you want to implement?
* [x] Did you follow other API endpoint implementations?
* [x] Did you cover it with test(s)?
* [x] Did you documented it in the [read me](https://github.com/emartech/suite-js-sdk/blob/master/README.md)?
* [x] Did you link the Emarsys Developer Hub?

### CODE REVIEW

File: api/endpoints/contactlist/index.js [hunk 1]
```diff
@@ -40,6 +40,23 @@ _.extend(ContactList.prototype, {
...
   },
 
+  fetch: function(payload, options) {
+    return this._requireParameters(payload, ['contact_list_id']).then(function() {
+      logger.log('contactlist_fetch');
+
+      var url = util.format('/v2/contactlist/%s/contactIds', payload.contact_list_id);
+
+      if (payload.next) {
+        url = util.format('/v2%s', payload.next);
```
Comment (eadaniel) [inline api/endpoints/contactlist/index.js:50]: `next` already has `$skipToken` and `$top` in it, i.e. you don't have to add it in line 55 for this case.

File: api/endpoints/contactlist/index.spec.js [hunk 2]
```diff
@@ -28,6 +28,28 @@ describe('SuiteAPI Contact List endpoint', function() {
...
+
+    testApiMethod(ContactListApi, 'fetch').withArgs({
+      contact_list_id: 2,
+      next: '/contactlist/330/contactIds?$skiptoken=750&$top=1000'
+    }).shouldGetResultFromEndpoint('v2/contactlist/330/contactIds?$skiptoken=750&$top=1000');
+
+    testApiMethod(ContactListApi, 'fetch').withArgs({
+      contact_list_id: 2,
+      next: null
+    }).shouldGetResultFromEndpoint('/contactlist/330/contactIds');
```
Comment (eadaniel) [inline api/endpoints/contactlist/index.spec.js:45]: Why is the contact list id `330` when `2` was specified in line 43?

File: api/endpoints/contactlist/index.spec.js [hunk 3]
```diff
@@ -28,6 +28,28 @@ describe('SuiteAPI Contact List endpoint', function() {
...
+
+    testApiMethod(ContactListApi, 'fetch').withArgs({
+      contact_list_id: 2,
+      $skiptoken: null
+    }).shouldGetResultFromEndpoint('/contactlist/330/contactIds?$skiptoken=750');
```
Comment (eadaniel) [inline api/endpoints/contactlist/index.spec.js:50]: Again: Why is the contact list id `330` when `2` was specified in line 43?

Why is there a `$skipToken` in the expected URL?

**Comment by eadaniel** [comment]
I would recommend to test the implementation with a test script against the real API on staging.

**Review by jpersson — APPROVED** [review]

**Review by akapa — APPROVED** [review]
takaros

**Pull request merged**